{"uuid":"f1d4968f-178c-4687-b59a-f09d3da13dfe","historyId":"9ee6e1d6c7258b6453ce5af5b6ceb4dc","fullName":"com.dsportal.qa.testcases.RegPageTest.RegisterValidUserTest","labels":[{"name":"package","value":"com.dsportal.qa.testcases.RegPageTest"},{"name":"testClass","value":"com.dsportal.qa.testcases.RegPageTest"},{"name":"testMethod","value":"RegisterValidUserTest"},{"name":"parentSuite","value":"Suite"},{"name":"suite","value":"Test"},{"name":"subSuite","value":"com.dsportal.qa.testcases.RegPageTest"},{"name":"host","value":"DESKTOP-UBR0902"},{"name":"thread","value":"351244@DESKTOP-UBR0902.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"RegisterValidUserTest","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [New Account Created. You are logged in as SDETAutomation9900] but found [password_mismatch:The two password fields didn’t match.]","trace":"java.lang.AssertionError: expected [New Account Created. You are logged in as SDETAutomation9900] but found [password_mismatch:The two password fields didn’t match.]\r\n\tat org.testng.Assert.fail(Assert.java:99)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1037)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:140)\r\n\tat org.testng.Assert.assertEquals(Assert.java:122)\r\n\tat org.testng.Assert.assertEquals(Assert.java:629)\r\n\tat org.testng.Assert.assertEquals(Assert.java:639)\r\n\tat com.dsportal.qa.testcases.RegPageTest.RegisterValidUserTest(RegPageTest.java:99)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\r\n\tat org.testng.TestNG.run(TestNG.java:1031)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","steps":[],"attachments":[{"name":"Page screenshot","source":"e0730388-7ba1-480d-bb77-c282e37dfc79-attachment","type":"image/png"},{"name":"RegisterValidUserTest failed and screenshot taken!","source":"15c242f0-8fd3-46b4-8958-6930c124ea35-attachment","type":"text/plain"}],"parameters":[],"start":1654798662340,"stop":1654798663053}